run-scripts/run-dann.py

```diff
@@ -0,0 +1,221 @@
+            command = base_command + ["--l-value-is", str(l_value_is)]
+
+            if i > 0 or run > 0:
+                # Remove conflicting --drop-old and --load flags
+                command = [arg for arg in command if arg not in ["--drop-old", "--load"]]
+                # Add skip flags if they are not already in the command
+                if "--skip-drop-old" not in command:
+                    command.append("--skip-drop-old")
+                if "--skip-load" not in command:
+                    command.append("--skip-load")
+
+            try:
+                random_number = random.randint(1, 100000)
+                print(f"Running command: {' '.join(command)}")
+                output_dir = f"results/pgdiskann/diskann/{case['db-label']}/{db_config['provider']}/{db_config['instance-type']}-{str(case['max-neighbors'])}-{str(case['l-value-ib'])}-{l_value_is}-{case['case-type']}-{run}-{random_number}"
+                os.environ["RESULTS_LOCAL_DIR"] = output_dir
+
+                os.makedirs(output_dir, exist_ok=True)
+
+                with open(f"{output_dir}/log.txt", 'w') as f:
+                    with redirect_stdout(f):
+                        print(f"Running benchmark: {benchmark_description}")
+                        print(f"DB Instance Type: {db_config['instance-type']}")
+                        print(f"DB Instance Provider: {db_config['provider']}")
+                        print(f"DB enable-seqscan: {db_config['enable-seqscan']}")
+                        for key, value in case.items():
+                            if key == "l-value-is":
+                                print(f"{key}: {l_value_is}")
+                            print(f"{key}: {value}")
+                        print("Current PostgreSQL configurations:")
+                        current_configs = query_configurations(db_config)
+                        for key, value in current_configs.items():
+                            print(f"{key}: {value}")
+                        print(f"Running command: {' '.join(command)}")
+                        f.flush()
+
+                    print("***********START***********")
+                    start_time = time.time()
+                    # Capture both stdout and stderr and write them to the log file
+                    subprocess.run(command, check=True, stdout=f, stderr=f)
+                    end_time = time.time()
+                    execution_time = end_time - start_time
+                    print(f"total_duration={execution_time}")
+                    print("***********END***********")
+                    f.flush()
+            except subprocess.CalledProcessError as e:
+                print(f"Benchmark failed: {e}")
+            print("Sleeping for 30 sec")
+            time.sleep(60)
+
+def main():
+    config = load_config("config.json")
+    start_time = time.time()
+    for case in config['cases']:
+        print(f"Running case: {case['db-label']}")
```

Comment:
@Sheharyar570 i think this is not the final version of the script, i don't see it running the queries in queries.json?